AnalysisMemoryTester build error

```
46>C:\Projects\GitHub\huguesv\PTVS\Python\Product\AnalysisMemoryTester\Program.cs(215,35,215,45): error CS1061: 'IPythonProjectEntry' does not contain a definition for 'UpdateTree' and no extension method 'UpdateTree' accepting a first argument of type 'IPythonProjectEntry' could be found (are you missing a using directive or an assembly reference?)
```

--- a/Python/Product/AnalysisMemoryTester/Program.cs
+++ b/Python/Product/AnalysisMemoryTester/Program.cs
@@ -212,7 +212,10 @@ namespace Microsoft.PythonTools.Analysis.MemoryTester {
                                 analyzer.LanguageVersion,
                                 new ParserOptions() { BindReferences = true }
                             );
-                            entry.UpdateTree(parser.ParseFile(), null);
+                            using (var p = entry.BeginParse()) {
+                                p.Tree = parser.ParseFile();
+                                p.Complete();
+                            }
                         }
                     }
                     break;

